website/payments/templates/payments/widgets/payment.html

```diff
@@ -5,7 +5,7 @@
     {% elif not payment and obj.payment_amount != 0 %}
         {% trans "Unpaid" %}
         -
-        € {{ obj.payment_amount }}
+        € {{ obj.payment_amount | floatformat:2}}
```

Comment:
```suggestion
        € {{ obj.payment_amount|floatformat:2}}
```

Not really necessary but I believe we format it like this everywhere.